client/src/components/EditProfile/useStyles.ts

```diff
@@ -0,0 +1,48 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+// eslint-disable-next-line
+const useStyles = (props: { isLessthanSm: boolean }) =>

```

Comment:
You can also use theme breakpoints instead of having to pass down a prop to check the screen size! https://material-ui.com/customization/breakpoints/#css-media-queries

Reply:
Will do that, Thank you